diagrams/onprem/network.py

```diff
@@ -89,5 +93,6 @@ class Zookeeper(_Network):
 ETCD = Etcd
 HAProxy = Haproxy
 OSM = OpenServiceMesh
+OPNSense = Opnsense
```

Comment:
You shouldn't add an alias by yourself in this file. Please read the contributing guide: https://github.com/mingrammer/diagrams/blob/master/CONTRIBUTING.md#update-aliases

Reply:
Doh! I didn't see that.  I think I resolved the concern. Thanks!